module/userProjects/helper.js

```diff
@@ -1226,38 +1238,70 @@ module.exports = class UserProjectsHelper {
 				let currentTask = project[0].tasks.find((task) => task._id == taskId)
 
 				let solutionDetails = currentTask.solutionDetails
-
 				let assessmentOrObservationData = {}
 
 				if (
-					project[0].entityInformation &&
-					project[0].entityInformation._id &&
+					// project[0].entityInformation &&
```

Comment:
@praveenKDass  check with @aks30 or @VISHNUDAS-tunerlabs 

in project as of now, we are not storing entityInformation. Hence always it will take from req body, for example if that project is an imp after completion observation, we need to store submission details in project and use that for creating observation. Verify this